### DISCUSSION TITLE
대시보드 삭제 확인 모달 (v)

### DISCUSSION THREAD

**Discussion by oceanlee-seoul** [Part3-Team15/taskify#108] [Ideas]
현재 피그마에서는 
대시보드 수정 페이지에서 대시보드 삭제를 눌렀을 때 
어떤 삭제에 대한 표시가 안나와있네요?!
이 부분도 컬럼 삭제 모달처럼 대시보드 삭제 모달 확인 모달 만드는 게 어떨까 싶습니다!!!

<img width="600" alt="스크린샷 2024-06-28 오후 12 46 13" src="https://github.com/Part3-Team15/taskify/assets/108586797/e38ff750-2560-407d-a36c-e515f8e41019">

다른 아이디어 있으시면 말씀해 주시고, 괜찮으시면 issue로 파겠습니다!

**Comment by un0211** [comment]
동의합니다! 카드는 몰라도 컬럼, 대시보드는 확인 있는게 좋을것 같아요~~
